lib/cli/src/commands/package/common/macros.rs

```diff
@@ -0,0 +1,79 @@
+macro_rules! make_pb {
+    ($self:ident, $msg:expr) => {{
+        let pb = indicatif::ProgressBar::new_spinner();
+        if $self.quiet {
+            pb.set_draw_target(indicatif::ProgressDrawTarget::hidden());
+        }
+
+        pb.enable_steady_tick(std::time::Duration::from_millis(500));
+        pb.set_style(
+            indicatif::ProgressStyle::with_template("{spinner:.magenta} {msg}")
+                .unwrap()
+                .tick_strings(&["✶", "✸", "✹", "✺", "✹", "✷"]),
+        );
+
+        pb.set_message($msg);
+        pb
+    }};
+
+    ($self:ident, $msg:expr, $($spinner:expr),+) => {{
+        let pb = indicatif::ProgressBar::new_spinner();
+        if $self.quiet {
+            pb.set_draw_target(indicatif::ProgressDrawTarget::hidden());
+        }
+
+        pb.enable_steady_tick(std::time::Duration::from_millis(500));
+        pb.set_style(
+            indicatif::ProgressStyle::with_template("{spinner:.magenta} {msg}")
+                .unwrap()
+                .tick_strings(&[$($spinner),+]),
+        );
+
+        pb.set_message($msg);
+        pb
+    }};
+}
+
+macro_rules! pb_ok {
+    ($pb:expr, $msg: expr) => {
+        $pb.set_style(
+            indicatif::ProgressStyle::with_template(&format!("{} {{msg}}", "✔".green().bold()))
+                .unwrap(),
+        );
+        $pb.finish_with_message(format!("{}", $msg.bold()));
+    };
+}
+
+macro_rules! pb_err {
+    ($pb:expr, $msg: expr) => {
+        $pb.set_style(
+            indicatif::ProgressStyle::with_template(&format!("{} {{msg}}", "✘".red().bold()))
+                .unwrap(),
+        );
+        $pb.finish_with_message(format!("{}", $msg.bold()));
+    };
+}
+
+macro_rules! bin_name {
+    () => {
+        match std::env::args().nth(0) {
+            Some(n) => n,
+            None => String::from("wasmer"),
+        }
+    };
+}
+
+macro_rules! cli_line {
+    () => {
+        std::env::args()
+            .filter(|s| !s.starts_with("-"))
+            .collect::<Vec<String>>()
+            .join(" ")
+    };
+}
```

Comment:
This doesn't need to be a macro. We should put this just in the one place where is being used

Reply:
https://github.com/wasmerio/wasmer/pull/4643/commits/64e969053f7124302e86fbb97c431abd86f99fb2#diff-171b04fbf128026d96a51378fbf46e7b2609e4aa69a86858705d394bb988fcb7L66